en/frames_multicopter/holybro_x500v2_pixhawk6c.md

```diff
@@ -0,0 +1,159 @@
+# Holybro X500 V2 + Pixhawk 6C (PX4 Dev Kit)
+
+This topic provides full instructions for building the [Holybro X500 V2 ARF Kit](http://shop.holybro.com/x500-v2-kit_p1288.html), also known as the Holybro PX4 Dev Kit.
+
+![The fully built vehicle with props removed](../../assets/airframes/multicopter/x500_v2_holybro_pixhawk6c/kit_no_props.jpg)
+
+
+## Assembly
+
+### Holders
+
+**Screw**-  Sunk Screw M2.5*6 12pcs
+
+1. Insert the hanger rubber ring gasket in each of their respective hangers.
+   Do not use sharp objects to press the rubbers inside.
+
+   @[youtube](https://youtu.be/4Tid-FCP_aI)
```

Comment:
FYI, you were using bullets. If the steps are in order, best to use numbers. Two things to note here:
1. You can always number using `1.`. Markdown will auto number for you.
2. If you indent to match the number, then the contents are part of the numbered item. So you get a nice indent on rendering.